source/iface/platform.rst

```diff
@@ -31,42 +36,58 @@ A platform contains 1 or more SYCL devices, or a host device.
 ::
 
   platform();
-  explicit platform(cl_platform_id platformID);
-  explicit platform(const sycl::device_selector &deviceSelector);
+  template <typename DeviceSelector> explicit platform(const DeviceSelector&);
 
 Construct a SYCL platform instance.
 
-The default constructor creates a host platform. When passed a
-``cl_platform_id``, an OpenCL|trade| platform is used to construct the
-platform. The ``cl_platform_id`` is retained and available via
-get_. When passed a :ref:`device_selector`, a platform is constructed
-that includes the preferred device.
+The default constructor create a SYCL platform instance that is a
+copy of the platform which contains the device returned by
+``default_selector_v``. When passed a :ref:`device selector<device-selectors>`,
+constructs a SYCL platform instance that is a copy of the platform which
+contains the device returned by the device selector parameter.
 
+===============
+Member function
+===============
 
-``get``
-=======
+``get_backend``
+===============
 
 ::
 
-  cl_platform_id get() const;
+  backend get_backend() const noexcept;
+
+Returns a backend identifying the SYCL backend associated with
+this platform.
+
+``get_info``
+============
 
-Returns the OpenCL device associated with the platform.
+::
 
-Only call this when the platform constructor was passed a
-``cl_platform_id``.
+  template <typename Param>
+  typename Param::return_type get_info() const;
 
-.. _platform-get_devices:
+Queries this SYCL platform for information requested by the template
+parameter ``Param``. The type alias ``Param::return_type`` must be defined in
+accordance with the info parameters in :ref:`device descriptors<device>`
```

Comment:
I think this is the wrong link?

Reply:
correct, this one is just temporary...after updating Devices chapter we will have real picture

Reply:
Shouldn't this be a link to the section describing "sycl::info::platform", which is on this same page?  Why does it depend on the device chapter?

Reply:
If we take a look into Specification get_info description from [Table 16](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.members.platform) :

_Queries this SYCL platform for information requested by the template parameter Param. The type alias Param::return_type must be defined in accordance with the info parameters in [Table 25](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.device.info) to facilitate returning the type associated with the Param parameter._

My understanding is that [Table 25](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.device.info) reflect Device information descriptors from Device class?

Reply:
Aha!  This is a bug in the specification that was already fixed in KhronosGroup/SYCL-Docs#407.  It's supposed to be a link to [Table 18](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.platform.info).

Unfortunately, that fix hasn't been published yet, which is why you don't see the correct link in the HTML spec.

Reply:
Fixed with commit : https://github.com/KhronosGroup/SYCL_Reference/pull/69/commits/e8124a86c3d1643546d1f4a4b3a57fa2220adcc2